imaids/cassettes.py

```diff
@@ -635,10 +650,14 @@ def create_radia_object(
                         self.nr_core_blocks/2),
                     self._end_blocks_length])
                 # is_pole_list if HYBRID and first core object is POLE
+                if self.terminations_poles:
+                    end_poles = [value if i % 2 == 0 else not value for i, value in enumerate(end_poles)]
+                    start_poles = end_poles[::-1]
```

Comment:
It seems that the start blocks are not simply the end blocks in reverse order.
For example, for a cassette with a pole as first non-termination item, we could have: ```_is_pole_list = [True, False, True, False, ..., True, False, True, False]```